core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/io/ImportResponse.java

```diff
@@ -84,21 +87,29 @@ public String getError() {
 	public Set<ComponentURI> getVisitedComponents() {
 		return visitedComponents;
 	}
+	
+	public int getChangeCount() {
+		return changeCount;
+	}
 
 	public static ImportResponse error(String error) {
-		return new ImportResponse(error, Set.of(), List.of());
+		return new ImportResponse(error, Set.of(), List.of(), 0);
 	}
 
 	public static ImportResponse success(Set<ComponentURI> visitedComponents) {
 		return success(visitedComponents, List.of());
 	}
 	
 	public static ImportResponse success(Set<ComponentURI> visitedComponents, List<ImportDefect> defects) {
-		return new ImportResponse(null, visitedComponents, defects);
+		return new ImportResponse(null, visitedComponents, defects, 0);
```

Comment:
This `0` can be a bit misleading on the UI side; the conditions in the PR do not prevent us from displaying e.g. `0 component(s) were imported, 99 items are displayed` when this factory method is called with a non-empty `visitedComponents` Set.